modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/internal/jaxrs/application/FDSApplication.java

```diff
@@ -309,10 +319,60 @@ public Response saveFDSCustomView(
 		).build();
 	}
 
+	private String _create(
```

Comment:
Hey @RenanUmV, just inline this method into its only usage. At the end , you will have something like this.

```
return Response.ok(
	_objectMapper.writeValueAsString(
		new FDSResponse(
			_getFDSTableRows(
				fdsDataProvider.getItems(
					fdsKeywordsFactory.create(httpServletRequest), fdsPagination, httpServletRequest, sort),
					tableName, httpServletRequest, groupId),
				fdsDataProvider.getItemsCount(
					fdsKeywordsFactory.create(httpServletRequest),httpServletRequest)),
					httpServletRequest),
				MediaType.APPLICATION_JSON
			).build();
```